packages/MarkdownEditor-Tests.package/MarkdownDelimiterEmphasisTest.class/instance/testBeginsWith.st

```diff
@@ -0,0 +1,7 @@
+tests - recognition
+testBeginsWith
```

Comment:
maybe put this into two test cases, one being testBeginsWithAsterisk and the other testDoesNotBeginWithOtherCharacter